templates/pricing/index.html

```diff
@@ -1,94 +1,123 @@
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Ubuntu Pro and support</h2>
       </div>
-      <div class="col-6 col-start-large-7">
-        <ul class="p-list">
+      <div class="col">
+        <p>
+          10 years of security maintenance for over 25,000 packages, kernel livepatch, landscape, FIPS certified modules, CIS and DISA-STIG profiles, and more.
+        </p>
+        <p>Optional telephone/email support for Ubuntu OS, infrastructure and application.</p>
+        <ul class="p-list--divided">
           <li class="p-list__item is-ticked">Ubuntu base OS</li>
           <li class="p-list__item is-ticked">Kernel</li>
           <li class="p-list__item is-ticked">Kubernetes</li>
           <li class="p-list__item is-ticked">OpenStack</li>
           <li class="p-list__item is-ticked">Docker</li>
-          <li class="p-list__item is-ticked">Storage</li>
+          <li class="p-list__item is-ticked">Ceph and Swift Storage</li>
           <li class="p-list__item is-ticked">Applications</li>
         </ul>
-        <ul class="p-inline-list u-no-margin--bottom">
-          <li class="p-inline-list__item"><a class="p-button--positive u-no-margin--bottom" href="/pro">Purchase Ubuntu Pro</a></li>
-          <li class="p-inline-list__item"><a class="p-button u-no-margin--bottom" href="/pricing/infra">See our services</a></li>
-        </ul>
+        <div class="p-cta-block">
+          <a class="p-button--positive" href="/pro">Purchase Ubuntu Pro</a>
+          <a class="p-button"
+             href="/pricing/infra"
+             aria-label="See our Ubuntu Pro infrastructure services">See our services</a>
+        </div>
       </div>
     </div>
-    <div class="u-fixed-width">
-      <hr style="margin: 2rem 0;" />
-    </div>
-    <div class="row">
-      <div class="col-5">
-        <h2 class="p-heading--3">Fully managed infrastructure</h2>
-        <p>Fully managed OpenStack and Kubernetes.</p>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Fully managed infrastructure</h2>
       </div>
-      <div class="col-6 col-start-large-7">
-        <ul class="p-list">
+      <div class="col">
+        <p>Fully managed OpenStack and Kubernetes.</p>
+        <ul class="p-list--divided">
           <li class="p-list__item is-ticked">Multi-cloud K8s</li>
           <li class="p-list__item is-ticked">Managed private cloud</li>
           <li class="p-list__item is-ticked">Data center automation</li>
           <li class="p-list__item is-ticked">Logging, alerting, monitoring</li>
         </ul>
-        <p class="u-no-margin--bottom"><a class="p-button u-no-margin--bottom" href="/pricing/infra#managed">Managed open infrastructure</a></p>
+        <div class="p-cta-block">
+          <a class="p-button"
+             href="/pricing/infra#managed"
```

Comment:
Not sure that this is in scope for this pr, but it looks like the "managed" id might have been removed on the /pricing/infra page as this link doesn't jump to any specific section when clicked